my_list.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('my_enumerable')
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] According to the project requirements, You were asked to implement In MyList class a method #each that yields successive members of @list and uses the MyEnumerable module. Kindly add this ✍🏻 
- Your code might look like this:

```ruby
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```